src/stages/normalize/patchers/FunctionApplicationPatcher.js

```diff
@@ -82,7 +82,17 @@ export default class FunctionApplicationPatcher extends NodePatcher {
     let { args } = this;
     let lastArg = args[args.length - 1];
     if (lastArg.isMultiline()) {
-      this.insert(this.contentEnd, `\n${this.getIndent()})`);
+      /**
+       * in case the next non-whitespace token is `.` we have to put the ) on it's line, so we lookahead.
```

Comment:
For comments inside functions, use `//` instead of `/*` for comments. Also, comments should start with a capital letter.

Also, maybe mention that we need to do this because the CoffeeScript compiler will sometimes reject `)` followed by `.` on a later line.